SS14.Launcher/Views/MainWindowTabs/OptionsTabView.xaml

```diff
@@ -11,12 +11,8 @@
         <mainWindowTabs:OptionsTabViewModel />
     </Design.DataContext>
 
-    <!--<Grid Margin="4" VerticalAlignment="Top" ColumnDefinitions="Auto, *">
-        <TextBlock Grid.Column="0" VerticalAlignment="Center" Text="Player Name:" Margin="4" />
-
-        <DockPanel Grid.Column="1">
-            <Button DockPanel.Dock="Right" Content="Save" Command="{Binding PlayerNameSaveButtonPressed}" />
-            <TextBox Margin="4" VerticalAlignment="Center" Text="{Binding PlayerName,Mode=TwoWay}" />
-        </DockPanel>
-    </Grid>-->
-</UserControl>
\ No newline at end of file
+    <StackPanel Orientation="Vertical">
+        <CheckBox Grid.Row="0" Grid.Column="0" VerticalAlignment="Center" Margin="4" IsChecked="{Binding ForceGLES2,Mode=TwoWay}">Compatibility Mode</CheckBox>
+        <TextBlock Grid.Row="1" Grid.Column="0" VerticalAlignment="Center" TextWrapping="Wrap" Text="This uses OpenGL ES 2 (via ANGLE if necessary), which is less likely to suffer from driver bugs. Use this if you are experiencing graphical issues." Margin="8" />
```

Comment:
You're not using a grid here, so...
```suggestion
        <CheckBox VerticalAlignment="Center" Margin="4" IsChecked="{Binding ForceGLES2,Mode=TwoWay}">Compatibility Mode</CheckBox>
        <TextBlock VerticalAlignment="Center" TextWrapping="Wrap" Text="This uses OpenGL ES 2 (via ANGLE if necessary), which is less likely to suffer from driver bugs. Use this if you are experiencing graphical issues." Margin="8" />
```

Reply:
...right. (In case you're wondering why it was done this way: It *was* a grid at one point.)